R/optparse.R

```diff
@@ -485,7 +491,26 @@ parse_args <- function(object, args = commandArgs(trailingOnly = TRUE),
                 options_list[[option@dest]] <- FALSE
             } else {    
                 options_list[[option@dest]] <- option_value
+            }         
+            if (option@action == "callback") {
+                if(!is.function(option@callback))
+                    stop(springf("callback not callable"))
+                if(option@callback_args != NULL && (!is.list(callback_args) && names(callback_args) != NULL)) 
+                    stop(sprinf("callback_args, if supplied, must be ordinary list"))
```

Comment:
I'm not sure if a non-named list is not an "ordinary list", more clear to call this an "unnamed" list

    "callback_args, if supplied, must be an unnamed list"